api/authentication.py

```diff
@@ -33,3 +33,16 @@ def authenticate_credentials(self, key):
         if not user.is_active:
             raise exceptions.AuthenticationFailed('User inactive or deleted.')
         return (user, token)
+
+
+class IsSecurityUser(BasePermission):
```

Comment:
this is better suited in `api/permissions.py`

Reply:
working on it